notes/v1.0.0-beta2.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+## Notable changes
+<!-- Add the list of notable changes, remove otherwise.-->
+This is a slim release on the SDK side, mainly contains bug fixes, version updates development improvements.
+As usual, the autogenerated Go code has been updated to match the 2.5.0 ECE API swagger specification.
+
+### Bug fixes
+<!-- Add the list of bug fixes, remove otherwise.-->
+
+#### Plan tracker reported some plans as successfully finished when they actually failed
+
+Fixes a bug in the plan tracker logic which made it return the first
```

Comment:
`more notably` --> `most notably`

Maybe change `set a cluster vacate as "Sucessful"` --> `flag a cluster vacate as sucessful` (if that doesn't change the meaning of the sentence)